scripts/generate-clojure-spec-cache.cljs

```diff
@@ -1,16 +1,7 @@
-(ns my.gen
+(ns generate-cljs-cache
```

Comment:
it's better to have namepspace with more than one segment
Otherwise you get this warning:
```
WARNING: generate-cljs-cache is a single segment namespace at line 1
```